plugins/ufw/appprofiles.cpp

```diff
@@ -77,19 +73,19 @@ const QList<Entry> & get()
             }
         qSort(profiles);
     }
-        
+
     return profiles;
 }
 
 Entry get(const QString &name)
 {
-    QList<Entry>::ConstIterator it(get().constBegin()),
-                                end(get().constEnd());
-
-    for(; it!=end; ++it)
-        if((*it).name==name)
-            return *it;
-    return Entry(QString());
+    // This feels *so* wrong.
```

Comment:
I've been away from this code for too long. At the point it feels written by someone else. Mind to explain what you find wrong here ?

Reply:
the `wrong` thing is in the code that I wrote, if you read it quickly you see that it's a `for(... : get()` - Forget. not a good thing to say when we iterate over a collection.